sql/moz-fx-data-marketing-prod/ga_derived/wnp_hits_v1/metadata.yaml

```diff
@@ -0,0 +1,19 @@
+description: This table aggregates the traffic to WNP pages and the number of bounces
+friendly_name: WNP hits and bounces
+labels:
+  incremental: true
+  schedule: daily
+owners:
+  - rbaffourawuah@mozilla.com
+scheduling:
+  dag_name: bqetl_google_analytics_derived
```

Comment:
Now that I am building the logic on www_site_hits_v1, do I still need to include this in the logic @sean-rose

Reply:
No, because [`www_site_hits_v1` itself has a dependency on `www_site_empty_check_v1`](https://github.com/mozilla/bigquery-etl/blob/c68a89b8711f18b11c6f8bb0e3e05768bb45bc00/sql/moz-fx-data-marketing-prod/ga_derived/www_site_hits_v1/metadata.yaml#L10-L11).